app/models/claim.rb

```diff
@@ -50,6 +49,9 @@ class Claim < ActiveRecord::Base
   before_update -> { secondary_claimants.destroy_all },
     if: :additional_claimants_csv_changed?
 
+  before_update :remove_additional_claimants_csv!,
+    if: -> (c) { c.secondary_claimants.any? }
```

Comment:
you're right although it's a lambda so it would still need an arity of 1 because it's called with the claim. could change it to:

```ruby
before_update :remove_additional_claimants_csv!,
  if: -> (_) { secondary_claimants.any? }
```

?